.circleci/docker/common/install_conda.sh

```diff
@@ -55,6 +55,10 @@ if [ -n "$ANACONDA_PYTHON_VERSION" ]; then
   # Ensure we run conda in a directory that jenkins has write access to
   pushd /opt/conda
 
+  # Prevent conda from updating to 4.14.0, which causes docker build failures
+  # Examples: https://hud.pytorch.org/pytorch/pytorch/commit/754d7f05b6841e555cea5a4b2c505dd9e0baec1d
+  echo "conda ==4.13.0" >> ~/conda-meta/pinned
+
   # Track latest conda update
   as_jenkins conda update -y -n base conda
```

Comment:
Err, why updating it at all then? Just use the bundled one...
```suggestion
  #  Use conda 4.13.0, as 4.14. causes docker build failures,
  #  See https://hud.pytorch.org/pytorch/pytorch/commit/754d7f05b6841e555cea5a4b2c505dd9e0baec1d
  # Uncomment when resolved
  # as_jenkins conda update -y -n base conda
```

Reply:
Cuz the prev conda installed is 4.12, but yea, that's a simpler change, will take suggestion.